src/classes/CON_PrimaryAffls_TEST.cls

```diff
@@ -768,51 +768,492 @@ private with sharing class CON_PrimaryAffls_TEST {
     ****************************************************** FUNCTIONAL TESTS ***************************************************
     **************************************************************************************************************************/
 
+
+    /**************************************************************************************************************************************
```

Comment:
For this class, do we want to consider using `Database.insert(List<SObject>,false)` and `SObject.getErrors()` to detect that the error is specifically added to each contact? It may be a more specific test and add value.